src/support/filename.c

```diff
@@ -48,3 +48,59 @@ __wt_nfilename(
 
 	return (0);
 }
+
+/*
+ * __wt_remove_if_exists --
+ *	Remove a file if it exists.
+ */
+int
+__wt_remove_if_exists(WT_SESSION_IMPL *session, const char *name)
+{
+	int exist;
+
+	WT_RET(__wt_exist(session, name, &exist));
+	if (exist)
+		WT_RET(__wt_remove(session, name));
+	return (0);
+}
+
+/*
+ * __wt_fp_open --
+ *	Open a FILE handle.
+ */
+int
+__wt_fp_open(WT_SESSION_IMPL *session, const char *name, FILE **fpp)
```

Comment:
I like this - but there are a decent number of places that currently call fopen/fclose that should use this now not included in this change, but should be.